docs/apis-tools/zeebe-api-rest/zeebe-api-rest-overview.md

```diff
@@ -0,0 +1,13 @@
+---
+id: zeebe-api-rest-overview
+title: "Overview"
+description: "The Zeebe REST API enables clients to communicate with the cluster. Activate jobs, cancel and create process instances, and more."
+---
+
+## Introduction
+
+The Zeebe REST API is a REST API designed to interact with the Zeebe process automation engine.
+
+:::note
+Ensure you authenticate before accessing the Zeebe REST API.
+:::
```

Comment:
This is lacking any details on how to authenticate! I'll follow up in #3277 about who will write this documentation.

Reply:
I will add this. We'll link to the same page as we did for the gRPC stuff, as it won't greatly differ.

Reply:
I missed this message @npepinpe! My apologies for doing something that you said you were going to do 😅

Reply:
All good :)